stable/autonity-network/templates/NOTES.txt

```diff
@@ -6,6 +6,12 @@ kubectl -n {{ .Release.Namespace }} get secrets account-pwd -o 'go-template={{ "
 # Get private key of first observer (id == 0)
 kubectl -n {{ .Release.Namespace }} get secrets observers -o 'go-template={{ "{{" }}index .data "0.private_key"{{ "}}" }}' | base64 --decode; echo ""
 
+# Get private key of Gvernance Operator
```

Comment:
Minor comment, but you typo'd governance here

Reply:
fixed